### DISCUSSION TITLE
Traceability in Argo Events.

### DISCUSSION THREAD

**Discussion by maganaluis** [argoproj/argo-events#1245] [Ideas]
We have an interesting use-case for Argo Events, and we are actually using along KNative which is part of Kubeflow. One of the things we are trying to standardize is on Traceability, KNatvie supports Zipkin and so does Dapr another KNative alike framework. 

Having multiple eventing frameworks is not necessarily a bad thing since a single framework does not have all the functionalities we are looking for, but only if we can trace events across them. I don't see any support for Zipkin at the moment in Argo. Would the community be open to add this feature? or receiving this as a PR?

**Comment by whynowy** [comment]
Traceability is one of the features we are looking for, and there's also a [request](https://github.com/argoproj/argo-events/issues/1111) from the community asking for OpenTelemetry support. However we are short of resources for it, you are very welcome to contribute the implementation if you would like!

**Comment by kaio6fellipe** [comment]
Hey guys, @whynowy @maganaluis. I know that it is a old discussion but just included a PR that adds OpenTelemetry support https://github.com/argoproj/argo-events/pull/3961.
